lua/rainbow-delimiters.lua

```diff
@@ -56,6 +59,5 @@ local M = {
 	toggle  = toggle,
 }
 
-
```

Comment:
I like that extra space before the last statement group. It gives me visual separation between the body of the script and the closing statements.

Reply:
I have added the space again. I think I accidentally removed it when removing the previously mentioned `__index` code.